CONTRIBUTING.md

```diff
@@ -109,12 +109,14 @@ CSS
 
 ```
 
-Here is a full list of the available CSS variables:
-
-- `--darkreader-neutral-background` should be mostly used for elements that have a wrong background color (usually bright backgrounds that should be dark).
-- `--darkreader-neutral-text` should be used for elements with a wrong text color (usually dark texts that should be light).
-- `--darkreader-selection-background` corresponds to user's Selection Background Color setting.
-- `--darkreader-selection-text` corresponds to user's Selection Text Color setting.
+Here is a full table of available CSS variables:
+
+| Variable                            	| Description                     	| Use                                                         	|
+|-------------------------------------	|---------------------------------	|-------------------------------------------------------------	|
+| `--darkreader-neutral-background`   	| Background Color                	| Mostly used for elements that have a wrong background color 	|
+| `--darkreader-neutral-text`         	| Text Color                      	| Used for elements with a wrong text color                   	|
+| `--darkreader-selection-background` 	| User's Text Color setting       	| The user's Text Color setting                               	|
+| `--darkreader-selection-text`        	| User's Background Color setting 	| The user's Background Color setting                         	|
```

Comment:
Any idea why the indent for `|` is not like the other ones?

Reply:
I think my formatter is having a seizure

Reply:
I think this is an issue on your side 
<img width="1907" alt="image" src="https://user-images.githubusercontent.com/66189242/110685729-81775d00-81d6-11eb-8bbc-fa829de50948.png">

Reply:
I will try to compact it more :]

Reply:
LGTM